### DISCUSSION TITLE
Some shared mind map examples

### DISCUSSION THREAD

**Discussion by macmarrum** [freeplane/freeplane#254] [Shared mind maps]
Further to the **MindMap show and tell** discussion during [Freeplane community online conference on Sunday, March 06, 2022 ](https://sourceforge.net/p/freeplane/discussion/758437/thread/c58087bf88/), this is to have a single place to share interesting mindMaps

**Comment by macmarrum** [comment]
My idea about a Trello-like/Kanban-like board, based on dark_nord_template.mm

[board_based_on_dark_nord_template.mm.zip](https://github.com/freeplane/freeplane/files/8223310/board_based_on_dark_nord_template.mm.zip)

![board_based_on_dark_nord_template](https://user-images.githubusercontent.com/7197131/157660149-e7e694ea-f741-470d-a3de-555c44e77dfe.svg)

**Reply by quickfold** [reply]
Am I correct that there is currently no way to make this map "vertical" (90 degree rotation) in FP?

**Comment by macmarrum** [comment]
It's possible to make it "vertical" by dragging each node to be positioned under its parent -- manual layout.
But I believe you have the automatic layout in mind. Then it's true: only "horizontal".

**Comment by MarCorrado** [comment]
This is a template inspired to certain elements (colors, background) of the "Professional" template layout of Xmind.
I made it for use in work setting.
<img width="840" alt="professional" src="https://user-images.githubusercontent.com/61862680/160462636-3b98be6d-7dc4-46ff-95e6-00ada18b8bc9.png">
[professional.zip](https://github.com/freeplane/freeplane/files/8365306/professional.zip)

**Comment by ofossum** [comment]
Made a template a while ago for having some "clean-sheet-of-paper" look but also a bit IT-like… so i came up with this.
Font is: [Iosevka Fixed](https://typeof.net/Iosevka/)

![code_bw](https://user-images.githubusercontent.com/99810909/163064581-699a2a5a-64aa-436a-8d4f-87dd0b1f1d44.png)

**Reply by rexelbartolome** [reply]
this looks neat!

**Reply by zapyon** [reply]
Would you provide the style for download? Would be appreciated very much. Thank you.

**Reply by ofossum** [reply]
Sure, here you go! ;)
And this is were you can find the font [Iosevka](https://typeof.net/Iosevka/).
[code_bw.mm.zip](https://github.com/freeplane/freeplane/files/9175806/code_bw.mm.zip)

**Reply by zapyon** [reply]
Thank you very much!

**Comment by ofossum** [comment]
This is a part of my bachelors thesis.
I used Freeplane to structure my categories for qualitative analysis.
As you can see, i used:

* _Node-Hierarchy_ for categories, sub-categories and snippets from interviews
* _Icons_ for separating the persons from my interviews (1 and 2)
  * _Attributes_ were used before, but were a lot more clickwork. 
    I figured out, that filtering for content from different persons could be done by filtering icons as well.
* _Attributes_ for time-codes from the according interviews, to have a quick way to look up things / re-hear the interviews
* _Details_ for some stuff i'd have to remember / what i mean by this

All in all it was really nice to use Freeplane: 
i was able so quickly change categories, annotate them and sorting in snippets from the interviews.
Also all informations were quickly accessible and could be filtered by icons or attributes.
Using the different levels was like using a spreadsheet-software, but much quicker and more intuitive (you have to sort like 1st level: main-category, 2nd-level: sub-category etc.).
And if necessary one could also embed images, use file-links for connect parts of writing…

![mindmap_categories_for_qualitative_analysis](https://user-images.githubusercontent.com/99810909/163067607-42d5e3f3-84e7-4365-9525-77bfb6167a05.png)

**Comment by ldmpub** [comment]
Hi all,

**Please find attached my "all in one" personal mind map template 😉**. Its' a kind of general project management Mind Map I'm using both for personal GTD or professional project lead.

![image](https://user-images.githubusercontent.com/1452146/164891543-7a7581d0-bca2-4c07-bfca-567e21fa183e.png)

**Benefits**
- Organized tasks list
- Easy progress monitoring trough icons and colors
- Keyboard shortcuts to quickly change node style / task status.

**The sample mind map attached contains 2 main branches**
* One branch with my original caption to present and explain the use of the '9' nodes style
* An other branch with sample nodes to illustrate the use of these styles in context while organizing a trip with friends.

Best regards,

[All in one Tasks Management - Libre Freeplane Mind Map Sample.zip](https://github.com/freeplane/freeplane/files/8546764/All.in.one.Tasks.Management.-.Libre.Freeplane.Mind.Map.Sample.zip)

**Comment by chminngi** [comment]
HI all,

I build within two days my own object/problem/knowledge  oriented domain link map with the latest Freeplane version under MacOS.
 
My first map is a **global map** [Img1] where all my tools, doc, links, shells are linked to the underlying MacOS, Chrome , secure shell extension, SSH handler, Firefox, VScode and iTerm features. Linux and Windows needs to be done and therefore has low priority. 

In this map I will not only collect all my  object/problem/knowledge  oriented  domain maps together ,even have a place to group all the development tools in one place.

A second map is a normal mind map called **test map** [Img2] linked to the **global map** for quick access and reference of global links or doc. Here I'll group all links and information that belongs to the object/problem/knowledge domain.

![00000000-Global](https://user-images.githubusercontent.com/15190791/199737484-f548bfb8-aaff-4a82-b7aa-2a8dadbcbb68.png)
[Img1: Global map]

![00000000-Test](https://user-images.githubusercontent.com/15190791/199737573-2086bf1d-f529-4b26-88cf-e27c3eacb2ca.png) 
[Img2: Test map]

Normal mind mapping starts on the right hand side of the root node called test in my test map. 
Of course there is a lot more to do, far to be ready. Hope you got the idea! Freeplane is awesome!
 
 Best regards,

**Reply by reregaga** [reply]
Very interesting, but useless for beginners without the attached your `.mm` file as the previous commentators in this thread did.

**Reply by chminngi** [reply]
Thanks @reregaga for your comment, of course if there is some interest I'll post my work here. I need to remove some sensitive data before doing that.
.. and need to provide a `.md` file for explaining how all tools are wired together.

**Reply by Foxxsayy** [reply]
I'm super interested in whatever you're using here. I especially love the sections.

**Reply by tukanos** [reply]
@reregaga I would be interested also.  The sections would make it a nice template.

**Comment by kenfhill84083** [comment]
Here is a map I am going to use for a Freeplane video tutorial of the Command Search feature.  It will be used in the video for a demo and it will serve as my script outline.  It uses some icons - don't know if they come thru with the map (in a zip file).  (I am unfamiliar with this new forum - I hope I'm doing things correctly.)
[command search (mm).zip](https://github.com/freeplane/freeplane/files/11357369/command.search.mm.zip)
![freeplane _ command search - video demo , script map](https://user-images.githubusercontent.com/6380301/235255921-900bb271-dcaf-45ef-9a95-5602d810d446.jpg)

**Reply by kenfhill84083** [reply]
I finally made my video : https://youtu.be/bxhCgYmTUJM

**Comment by Coffee-Search** [comment]
I have been testing the visualisation possibilities of freeplane.
With a couple of techniques revolving around free positioned nodes a lot of things are possible.
There is some scope for automation from dia, ical, bpmn, UML, etc. to freeplane - I might look in to one or more of these if I can identify a useful need.
The example file contains a few examples but I have fully documented how I created the Weekly Planner - from this you should have all the techniques to fathom the rest. I would not say this is intuitive but once you have the techniques it's not too hard-going to produce / manage diagrams in freeplane.  
UPDATED: Added axis to parent the day columns. Added a few more experiments and examples. 
[freeplane_examples_v2.zip](https://github.com/user-attachments/files/17431387/freeplane_examples_v2.zip)

![Screenshot from 2024-10-17 12-28-41](https://github.com/user-attachments/assets/6fb0a660-67b9-4c20-811c-5fd927389f64)

**Reply by guoranjun2** [reply]
very impressive!

**Reply by nnako** [reply]
Hi @Coffee-Search , very impressive, indeed. :-)

You mentioned an idea within your attached mindmap:

> It is feasible that a add-on can be developed that would import a week from something like ical and create a weekly planner node

And, indeed, you could to that already. For coupling any kind of other file structure like `ical`, or even other task managing applications like `jira`, you could use the [`freeplane-python-io`](https://github.com/freeplane/freeplane/discussions/697) library. The only thing missing would be a kind of file structure translator implemented in Python.

As a preparation, I analyzed how your weekly planner would represent itself in the Freeplane mindmap `XML` file (mm), which is the basis on which the `freeplane-python-io` operates. Here is just a "teaser" to see the relevant XML structure elements of the "Mon" column including the elements displayed within your post:

![image](https://github.com/user-attachments/assets/5112cd79-b7c1-4711-87a1-99f80985b4d5)

The whole structure representation will be found in the attached mindmap.

[weekly_planner__xml-structure.zip](https://github.com/user-attachments/files/17449369/weekly_planner__xml-structure.zip)

I hope that we will continue on this... ;-)

**Comment by Coffee-Search** [comment]
Update to my visualisation examples mindmap  
[freeplane_examples_v3.zip](https://github.com/user-attachments/files/17632498/freeplane_examples_v3.zip)
Added ...
Diagram examples
Chart (line/bar) examples
Visualisation examples
HTML formatting examples including formulas
Embedded images using base64 - python code and examples
Tips/Howto

![Screenshot from 2024-11-05 11-43-00](https://github.com/user-attachments/assets/fc5a73bf-6417-4f2b-9961-1c7d74787581)
![Screenshot from 2024-11-05 11-43-43](https://github.com/user-attachments/assets/1d9c9631-71bf-48c2-a530-dd108287cc3f)
![Screenshot from 2024-11-05 11-44-13](https://github.com/user-attachments/assets/77537203-6a6d-45df-9911-2e165992e12b)
![Screenshot from 2024-11-05 11-44-55](https://github.com/user-attachments/assets/85d3a290-6fed-4240-ae8a-94090b045ec0)
![Screenshot from 2024-11-05 11-45-22](https://github.com/user-attachments/assets/9db93521-f0fe-4894-92af-469560367e89)
![Screenshot from 2024-11-05 11-45-58](https://github.com/user-attachments/assets/ea4fada1-d1a1-4384-9e52-34aafdcac1e4)
![Screenshot from 2024-11-05 11-46-38](https://github.com/user-attachments/assets/3f46fed4-db1d-428d-8474-a8266955f4c5)
![Screenshot from 2024-11-05 11-49-05](https://github.com/user-attachments/assets/1e3a2a26-fe58-4a99-8831-5de0e2960310)
![Screenshot from 2024-11-05 12-13-07](https://github.com/user-attachments/assets/db017c4a-d47c-406a-a39e-f9ca7569d78e)
![Screenshot from 2024-11-05 12-24-38](https://github.com/user-attachments/assets/38a4f6c7-09f6-4bff-9a1f-4fa85da79715)

**Reply by guoranjun2** [reply]
I’ve got to say, you have quite the imagination.

**Reply by nnako** [reply]
Hi @Coffee-Search , you impress me!!! Wonderful. Loads of further ideas ...

**Reply by pedrofiesta** [reply]
I loved your idea for embedding images in the mm file using the html view.
I do not like that when pasting images, it puts the files in a subfolder.
I will explore a way of creating an add-on that will do the image conversion and html embedded when pasting it on a node.

**Comment by aaa1386** [comment]
I have created two types of templates in the Zira file. What do you think?

[CLOUD .zip](https://github.com/user-attachments/files/20156113/CLOUD.zip)
---
![image](https://github.com/user-attachments/assets/dc1e1451-9141-4220-becb-cf21e60f3ce5)

**Comment by BobLangill** [comment]
This mindmap is the one I use to track my personal projects.  The anchor links shown and included in the mind map are dead because they  point to mind maps that are stored locally on my computer.

![FreeplanePersonProjectMindmap](https://github.com/user-attachments/assets/539aa7a5-45a0-43f0-92f0-78c1a9e9101a)
[Personal Projects.zip](https://github.com/user-attachments/files/22995389/Personal.Projects.zip)
The mind map in the zip file contains 441 nodes. 

I thought a view of my conventions and techniques might be more useful than the list of my projects. I've added the expanded view of that branch.

![PersonProjects_conventions](https://github.com/user-attachments/assets/600fcfbd-eded-4fc5-ae04-5782707c4adf)

**Comment by spicedreams** [comment]
Here's my "Minutes+Dialog" template, used in real time on a shared screen to capture a discussion and turn it into agreement (at least about where the discussion went) and Minutes. The right half is for the dialog, the left half for formal minutes. You can of course just use one half or the other, depending on context. 

<img width="1513" height="915" alt="image" src="https://github.com/user-attachments/assets/8f3988a1-9e37-4748-b1b6-32b9c14d0bc1" />

Dialogue part: normally keep the Question, Evidence and Issue trees collapsed and make a copy of any of them as appropriate to work on. The content of each tree is again a set of template nodes to be copied and the template content replaced with actual content (keeping the icons- if you and the context like icons). 

Minutes part: the Meeting Template tree just captures the formal "top and tail" of recording a meeting. The Listening Cycle is a reminder to the chair, minute-taker and others how a meeting should reach concensus on a participant's view on something. The Decision Template tree is a set of reminders of the kind of decisions a meeting can take including delegating an action to someone. It is overall a reminder that the principal purpose of a formal meeting is to make decisions.

**Reply by spicedreams** [reply]
here's the template itself- hope it's useful
[Minutes + Dialogue Template.zip](https://github.com/user-attachments/files/31496751/Minutes.%2B.Dialogue.Template.zip)
